<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SjMainWindow</class>
 <widget class="QMainWindow" name="SjMainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Safejumper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>375</width>
      <height>89</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #ffffff;
	border: 0;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="L_Version">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>71</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
color: #d6d6d6;
padding: 0;
margin: 0;
}</string>
     </property>
     <property name="text">
      <string>V 1.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="L_Logo">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>17</y>
       <width>188</width>
       <height>56</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-image: url(:/imgs/logo1.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameBottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>111</y>
      <width>375</width>
      <height>191</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>247</red>
          <green>247</green>
          <blue>247</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #f7f7f7;
	border: 0;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="b_Login">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>127</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/b-login-norm.png);
}
QPushButton:hover:!pressed
{
	border-image: url(:/imgs/b-login-hover.png);
}
QPushButton:pressed
{
	border-image: url(:/imgs/b-login-pressed.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="eLogin">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>8</y>
       <width>300</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-5</number>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius: 3px;
padding: 2px 30px 2px 15px;
border: 0 transparent #b8d4e9;
color: #444444;
border-image: url(:/imgs/e-inactive.png);
}
QLineEdit:focus
{
border-image: url(:/imgs/e-active.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ePsw">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>67</y>
       <width>300</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-5</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius: 3px;
padding: 2px 30px 2px 15px;
border: 0 transparent #b8d4e9;
color: #444444;
border-image: url(:/imgs/e-inactive.png);
}
QLineEdit:focus
{
border-image: url(:/imgs/e-active.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string notr="true">Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="lLoginDude">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>24</y>
       <width>14</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-image: url(:/imgs/login-dude.png);
}</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lPswLock">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>84</y>
       <width>14</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border-image: url(:/imgs/password-lock.png);
}</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cb_Rememberme">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>128</y>
       <width>100</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/cb-rememberme.png);
}
QPushButton:checked
{
	border-image: url(:/imgs/cb-rememberme-checked.png);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="b_options">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>150</y>
       <width>65</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/b-options.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>375</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="line_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>375</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>119</y>
       <width>375</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_Cancel">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>127</y>
       <width>110</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	color: #ffffff;
	border-image: url(:/imgs/b-cancel-norm.png);
}
QPushButton:hover:!pressed
{
	border-image: url(:/imgs/b-cancel-hover.png);
}
QPushButton:pressed
{
	border-image: url(:/imgs/b-cancel-pressed.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>b_Cancel</zorder>
    <zorder>b_Login</zorder>
    <zorder>eLogin</zorder>
    <zorder>ePsw</zorder>
    <zorder>lLoginDude</zorder>
    <zorder>lPswLock</zorder>
    <zorder>cb_Rememberme</zorder>
    <zorder>b_options</zorder>
    <zorder>line_2</zorder>
    <zorder>line_1</zorder>
    <zorder>line_3</zorder>
   </widget>
   <widget class="QFrame" name="frameHead">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #ffffff;
	border: 0;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>21</y>
       <width>375</width>
       <height>2</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	border:0px;
	border-image: url(:/imgs/h-line-mb.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_Cross">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>6</y>
       <width>10</width>
       <height>10</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/b-cross.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="L_Caption">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>3</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Safejumper</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_Minimize">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>6</y>
       <width>10</width>
       <height>10</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/b-min.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="b_Head">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>375</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border:0px;
	border-image: url(:/imgs/cb-empty.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>L_Caption</zorder>
    <zorder>b_Head</zorder>
    <zorder>b_Cross</zorder>
    <zorder>line_4</zorder>
    <zorder>b_Minimize</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>b_Login</sender>
   <signal>clicked()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>ToScr_Connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_options</sender>
   <signal>clicked()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>ToScr_Options()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_Rememberme</sender>
   <signal>toggled(bool)</signal>
   <receiver>SjMainWindow</receiver>
   <slot>cbRememberMe_Togg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Minimize</sender>
   <signal>clicked()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>Clicked_Min()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Head</sender>
   <signal>pressed()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>Pressed_Head()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Cross</sender>
   <signal>clicked()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>Clicked_Cross()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>SjMainWindow</receiver>
   <slot>Clicked_b_Cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
